pkg/app/destroy.go

```diff
@@ -32,9 +32,11 @@ var CommandDestroy = &cli.Command{
 	Usage:   "destroys the envd environment",
 	Flags: []cli.Flag{
 		&cli.PathFlag{
-			Name:    "path",
-			Usage:   "Path to the directory containing the build.envd",
-			Aliases: []string{"p"},
+			Name:        "path",
+			Usage:       "Path to the directory containing the build.envd",
+			Aliases:     []string{"p"},
+			DefaultText: ".",
+			Value:       ".",
```

Comment:
When --name is specified, there will be an error since both flags have values.

Reply:
That's a problem, I have fixed it by assigning the default value to `path` when both flags don't have values.